### ISSUE TITLE
As a PR reviewer, I'd like to know if anyone else (name and number of people) was assigned a review

### ISSUE THREAD

**Issue by ashanbrown** [pullreminders/backlog#114]
When I receive a review request and I'm the only reviewer, I feel more compulsion to review it quickly.  If others are listed as reviewers and I know they have more context than I do, I know I can leave the review to them.  I think there is already text that varies based on whether I've been included only as part of a team, but quickly seeing the number of other reviewers and their names (maybe other group names too) would be ideal.

**Comment by abinoda** [comment]
@ashanbrown Would you like this metadata shown on reminders or the real-time notifications, or both?

**Comment by ashanbrown** [comment]
I was thinking about in a DM but I could imagine it would be generally useful.

**Comment by ashanbrown** [comment]
On a similar tool I built I'd explicitly tell the user they were the "only" reviewer.

**Comment by abinoda** [comment]
@ashanbrown For DMs, there are two types of notifications – the real-time alerts and the scheduled reminders. I was asking which of these notifications types you'd like the additional context for. Either or both?

**Comment by ashanbrown** [comment]
Both sounds good.

**Comment by abinoda** [comment]
Another user requested adding PR size as context to the messages as well.

**Comment by ashanbrown** [comment]
Yeah, that sounds good.  It's another thing I'd take into account about
whether to review a PR immediately.

**Comment by abinoda** [comment]
From another user:

> It will be nice to add to this message a bit more info like number of files changed, number of added/deleted/modified raws to give whom ever gets the message a notion on the complexity of the PR. Very small PRs can be reacted almost immediately. Very large PRs can be disregarded almost immediately on a busy period. All the rest can be open for negotiation.
